config/rbac/hive_admin_role_binding.yaml

```diff
@@ -7,6 +7,8 @@ metadata:
   name: hive-admin
 roleRef:
   name: hive-admin
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
The hive-admin and hive-reader ClusterRoleBindings should not be created when running on kind. Are you seeing them created when running on kind?

Reply:
Nop, I don't think so:

```
kustomize build overlays/deploy | kubectl apply -f -
namespace/hive created
serviceaccount/hive-operator created
clusterrole.rbac.authorization.k8s.io/hive-frontend created
clusterrole.rbac.authorization.k8s.io/hive-operator-role created
clusterrole.rbac.authorization.k8s.io/manager-role created
clusterrole.rbac.authorization.k8s.io/system:openshift:hive:hiveadmission created
rolebinding.rbac.authorization.k8s.io/extension-server-authentication-reader-hiveadmission created
rolebinding.rbac.authorization.k8s.io/hiveapi created
clusterrolebinding.rbac.authorization.k8s.io/auth-delegator-hiveadmission created
clusterrolebinding.rbac.authorization.k8s.io/hive-frontend created
clusterrolebinding.rbac.authorization.k8s.io/hive-operator-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/hiveadmission-hive-hiveadmission created
clusterrolebinding.rbac.authorization.k8s.io/hiveapi-cluster-admin created
clusterrolebinding.rbac.authorization.k8s.io/manager-rolebinding created
deployment.apps/hive-operator created
rm -rf overlays/deploy
```

But even if is not loaded, the kubectl validator gives you an error ://.